src/haz3lweb/www/style.css

```diff
@@ -442,7 +441,43 @@ select {
   transform: scale(100%);
   cursor: not-allowed;
 }
-
+#top-bar .siblings {
+  transform:translate(0%, 50%);
+  direction: rtl;
+  position: relative;
+  bottom: -1.5em;
+  left: -1.2em;
+  max-height: 10em;
+  overflow: auto;
+  display: none;
+  flex-direction: column;
+  align-items: flex-end;
+  padding-left: 0.1em;
+  padding-right: 0.5em;
+  white-space: nowrap;
+  border-radius: 0 0.2em 0em 0;
```

Comment:
is the rounded edge on just one corner intentional?

Reply:
I just copy it from context-inspector where the top-right corner is the only corner not close to other frame. So here maybe I should rounded two bottom corner.